.github/workflows/publish-utbot.yml

```diff
@@ -57,6 +57,10 @@ jobs:
         run: |
           chmod +x docker/action-scripts/prepare-ubuntu-release.sh
           ./docker/action-scripts/prepare-ubuntu-release.sh
+      - name: auto installation check
+        run: |
```

Comment:
Move run to separate job, for run on clean runners

Reply:
@ladisgin
Look, for now, we’re going to test installation for each action being started via the publish job, and artifacts are part of a particular execution (means, the only way we can get them is to rely on working directory/dev env). If a check fails, we cannot go further with that publishing, users won’t get an unstable version.

However, if the final goal is to repeat the end-to-end installation scenario for each build (including ones being triggered for PRs), we must establish a staging zone for artifacts first. Otherwise, it’s not obvious where can we get artifacts for a particular build we’re going to check on against a clear env. Do we already have a staging in GitHub?

Reply:
We need test installation In separate job because we need check, that all dependency in archive.
For run job after publish add:
`needs: publish`
For download artifact:
```
- name: Download a single artifact
  uses: actions/download-artifact@v2
  with:
    name: my-artifact
```